core/scripts/vrfv2/testnet/super_scripts.go

```diff
@@ -23,13 +24,14 @@ func deployUniverse(e helpers.Environment) {
 	// required flags
 	linkAddress := deployCmd.String("link-address", "", "address of link token")
 	linkEthAddress := deployCmd.String("link-eth-feed", "", "address of link eth feed")
-	subscriptionBalanceString := deployCmd.String("subscription-balance", "", "amount to fund subscription")
+	subscriptionBalanceString := deployCmd.String("subscription-balance", assets.Ether(10).String(), "amount to fund subscription")
 
 	// optional flags
-	fallbackWeiPerUnitLink := deployCmd.String("fallback-wei-per-unit-link", "60000000000000000", "fallback wei/link ratio")
+	fallbackWeiPerUnitLink := deployCmd.String("fallback-wei-per-unit-link", assets.GWei(60000000).String(), "fallback wei/link ratio")
```

Comment:
You can use scientific notation or add underscores to make reading these long integer literals easier. In this case, either `60_000_000` or `60e6` would be easier to read than `60000000` (the former makes it clear that you're saying 60 million gwei, the latter I'd have to count the zeros one by one).